conf/app.routes

```diff
@@ -89,3 +89,6 @@ GET        /:arrivalId/report
 POST       /:arrivalId/report                                                                                        controllers.OtherThingsToReportController.onSubmit(arrivalId: ArrivalId, mode: Mode = NormalMode)
 GET        /:arrivalId/change-report                                                                                 controllers.OtherThingsToReportController.onPageLoad(arrivalId: ArrivalId, mode: Mode = CheckMode)
 POST       /:arrivalId/change-report                                                                                 controllers.OtherThingsToReportController.onSubmit(arrivalId: ArrivalId, mode: Mode = CheckMode)
+
+GET        /:arrivalId/do-you-have-anything-else-to-report                                                           controllers.DoYouHaveAnythingElseToReportYesNoController.onPageLoad(arrivalId: ArrivalId, mode: Mode = NormalMode)
```

Comment:
should this be `/:arrivalId/add-report` ?

Reply:
Ok done